Add ability to expose devices by label in google_assistant integration (#43100)
home-assistant/home-assistant.github.io: source/_integrations/google_assistant.markdown

  description: List of entity domains to expose to Google Assistant if `expose_by_default` is set to true. This has no effect if `expose_by_default` is set to false.
  required: false
  type: list
exposed_labels:

>   description: List of labels, where entities having any of these labels will be exposed.

Reviewer: ```suggestion
  description: A list of labels; entities assigned any of these labels will be exposed.
```
